src/main.jsx

```diff
@@ -4,17 +4,19 @@ import './styles/main'
 
 import React from 'react'
 import { render } from 'react-dom'
-import { I18n } from './lib/I18n'
 
-import App from './components/App'
+const renderApp = function () {
+  const Root = require('./components/Root').default
+  render(<Root />, document.querySelector('[role=application]'))
+}
 
-const context = window.context
-const lang = document.documentElement.getAttribute('lang') || 'en'
+if (__DEV__) {
+  window.React = React
+  require('preact/devtools')
```

Comment:
Ok, so doing it in `webpack.config.hot-reload.js` instead of conditioning it in `main,jsx` makes more sense IMHO. We can provide `React` with:

```js
plugins: [
  new webpack.ProvidePlugin({
    "window.React": "react"
  })
]
```

and force loading `devtools`:

```js
entry: [
  'webpack/hot/dev-server',
  'preact/devtools'
]
```

and so remove the now useless `__DEV__` compilation variable.

Whatever, it's just optimization. Thanks a lot for the PR :+1: